controllers/hibp.js

```diff
@@ -89,7 +89,7 @@ async function notify (req, res) {
     // or from "id" property (if subscribers record)
     const subscriberId = recipient.subscriber_id || recipient.id;
     const { recipientEmail, breachedEmail, signupLanguage, preFxaSubscriber } = getAddressesAndLanguageForEmail(recipient);
-    const campaignId = (preFxaSubscriber) ? "preFXA-see-all-breaches" : "see-all-breaches";
+    const campaignId = "go-to-dashboard-link";
```

Comment:
Can we change the utm params so that all of these are in the same campaign, but with different content?

E.g., on the "Go to dashboard" link ...

`utm_campaign=breach-resolution&utm_source=fx-monitor&utm_medium=email&utm_content=go-to-dashboard`

And then on the "Resolve this breach" link ...

`utm_campaign=breach-resolution&utm_source=fx-monitor&utm_medium=email&utm_content=resolve-this-breach`